pytest/tests/sanity/split_storage.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+"""
+Spins up an archival node with cold store configured and verifies that blocks are copied from hot to cold store.
+"""
+
+import sys
+import pathlib
+import os
+import copy
+import json
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[2] / 'lib'))
+
+from utils import poll_blocks
+from cluster import init_cluster, spin_up_node, load_config, get_config_json, set_config_json
+from configured_logger import logger
+
+
+def main():
+    config = load_config()
+    near_root, [node_dir] = init_cluster(1, 0, 1, config, [], {})
+
+    node_config = get_config_json(node_dir)
+
+    node_config["archive"] = True
+    # Need to create a deepcopy of the store config, otherwise
+    # store and cold store will point to the same instance and
+    # both will end up with the same path.
+    node_config["cold_store"] = copy.deepcopy(node_config["store"])
+    node_config["store"]["path"] = os.path.join(node_dir, 'data')
+    node_config["cold_store"]["path"] = os.path.join(node_dir, 'cold_data')
+
+    set_config_json(node_dir, node_config)
+
+    node = spin_up_node(config, near_root, node_dir, 0, single_node=True)
```

Comment:
we should also have a test, where we have 4 validators and one gets killed - this would create nice 'gaps' in the chain (as some blocks will be missing then)